lib/mangoex/client.ex

```diff
@@ -18,10 +18,18 @@ defmodule Mangoex.Client do
     GenServer.call(@client_name, {:create_user, body})
   end
 
+  def create_user(type, body) do
+    GenServer.call(@client_name, {:create_user, body, type})
+  end
+
   def update_user(user_id, body) do
     GenServer.call(@client_name, {:update_user, user_id, body})
   end
 
+  def update_user(type, user_id, body) do
+    GenServer.call(@client_name, {:update_user, user_id, body, type})
```

Comment:
Again, I think `user_id, body, type` are not in the order the related `handle_calls` expect.